.github/workflows/scripts/scanner-get-released-tags.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+#
+# Print a JSON containing all scanner vulnerability bundle streams, as directed
+# by the `VULNERABILITY_BUNDLE_VERSION` file.
+
+set -euo pipefail
+
+# Initialize the version map and json_array
+declare -A version_map
+json_array=()
+
+# Populate the version_map with version-to-tag mappings
+while read -r ver tag _; do
+    version_map[$ver]="$tag"
+done < <(./.github/workflows/scripts/scanner-output-release-versions.sh | jq -r '.versions[] | "\(.version) \(.tag)"')
+
+# Read the versions and their corresponding tags from the VULNERABILITY_BUNDLE_VERSION file.
+while read -r line; do
+    # Skip lines that are comments or empty
+    echo "$line" | grep -qE '^\s*(#.*|$)' && continue
+
+    read -r version ref <<< "$line"
+    ref=$(echo "$ref" | xargs)
```

Comment:
The `read` above should have stripped whitespace around the words, so this is superfluous.